package com.mankovskaya.githubtest.data.network.error

class ConnectionError : Exception()